modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/containers/setup-forms/SetupAnalyticsCloudForm/AdminInputs/index.js

```diff
@@ -0,0 +1,89 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+
+import {useLazyQuery} from '@apollo/client';
+import ClayForm, {ClayInput} from '@clayui/form';
+import {useEffect, useState} from 'react';
+import {Input} from '../../../../components';
+import useDebounce from '../../../../hooks/useDebounce';
+import {getBannedEmailDomains} from '../../../../services/liferay/graphql/queries';
+import {isValidEmail} from '../../../../utils/validations.form';
+
+const AdminInputs = ({admin, id}) => {
```

Comment:
should be admin the right name for this component?

Reply:
Yeah, this file doesn't exist anymore. I'm going to push again. Thank you!!